index.js

```diff
@@ -49,37 +49,40 @@ class AddressFinder extends React.Component {
     const { chosenAddress, results, error, searchTerm } = this.state;
     return (
       <div>
-        <div>Address finder:</div>
+        <label htmlFor="search">Address finder:</label>
         <div className="input_box">
           <input
+            id="search"
+            name="search"
             className="input"
             value={searchTerm}
             onChange={this.handleChange}
           />
           {searchTerm.length !== 0 && (
-            <div
+            <button
```

Comment:
`button`s are interactive and therefore tabbable, which is essential for keyboard only users.